app/_locales/en/messages.json

```
@@ -489,6 +489,9 @@
   "attributions": {
     "message": "Attributions"
   },
+  "auroraDeprecationWarning": {
+    "message": "MetaMask will be deprecating support for Aurora network"
+  },
```

would it be more useful to add a universal message for the upcoming network we want to deprecate?
```suggestion
  "auroraDeprecationWarning": {
    "message": "MetaMask will be deprecating support for $1 network"
  },
```

---

Updated here https://github.com/MetaMask/metamask-extension/pull/21933/commits/14a11f2cdb30b03e7437a1bf3eb685e135d8ff58